packages/components/src/internal/components/user/actions.ts

```diff
@@ -229,3 +232,14 @@ export function getUserSharedContainerPermissions(): Promise<string[]> {
         });
     });
 }
+
+export async function getUserGroups(userid: number): Promise<string[]> {
```

Comment:
Should this be taking in containerPath prop for subfoldering purposes?

Reply:
Only if we are going to be calling it for a folder other than the one we are currently in, otherwise selectRows will query the current container. So for example if I'm in `FolderA` but I want to call `getUserGroups` for `FolderB`, then we should add container path as an arg.